### PR TITLE
lang: [nl] Add Dutch Languages

### PR DESCRIPTION
**Pull request by snelledre**
Added the Dutch languages

### CODE REVIEW

**Comment by datamweb** [comment]
Hello, thanks for the PR.

We have [failure in PHPCSFixer](https://github.com/codeigniter4/shield/actions/runs/4502108787/jobs/7923519096?pr=691), can you run the following command to fix the error?
```console
composer cs-fix tests/Language/AbstractTranslationTestCase
```

File: src/Language/nl/Auth.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    'errorPasswordCommon'       => 'Het wachtwoord mag geen algemeen wachtwoord zijn.',
+    'suggestPasswordCommon'     => 'Het wachtwoord is vergeleken met meer dan 65.000 veelgebruikte wachtwoorden of wachtwoorden die via hacks zijn gelekt.',
+    'errorPasswordPersonal'     => 'Wachtwoorden mogen geen gehashte persoonlijke informatie bevatten.',
+    'suggestPasswordPersonal'   => 'Variaties op uw e-mailadres of gebruikersnaam mogen niet worden gebruikt voor wachtwoorden.',
+    'errorPasswordTooSimilar'   => 'Wachtwoord lijkt te veel op de gebruikersnaam.',
+    'suggestPasswordTooSimilar' => 'Gebruik geen delen van uw gebruikersnaam in uw wachtwoord.',
+    'errorPasswordPwned'        => 'Het wachtwoord {0} is openbaar gemaakt vanwege een datalek en is {1, aantal} keer gezien in {2} gecompromitteerde wachtwoorden.rds.',
+    'suggestPasswordPwned'      => '{0} mag nooit als wachtwoord worden gebruikt. Als u het ergens gebruikt, verander het dan onmiddellijk.',
+    'errorPasswordEmpty'        => 'Een wachtwoord is vereist.',
+    'errorPasswordTooLongBytes' => 'Password cannot exceed {param} bytes in length.',
```
[the remark this answers is not in the document]
Comment (snelledre) [reply src/Language/nl/Auth.php]: Sorry, I forget line 64 oeps.

**Comment by snelledre** [comment]
Sorry, I will try again today to update the previous one as well, but unfortunately I can't at the moment verify the other ones.
Probably due to ignorance as this is my first time on github to help with another project

**Comment by kenjis** [comment]
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits

**Comment by snelledre** [comment]
> It seems your PR branch is a bit out of dated.
> 
> Try first:
> 
> ```
> $ git fetch upstream 
> $ git rebase upstream/develop
> ```

Sorry, but that didn't help the output stays the same.
The repo is in sync i'm sure.
With the time from yesterday I'm trying now an hour off 3 with no results.
I don't understand why so small thing is so hard to get right.
I want to help with the dutch translation but not giving up right now so do you have some other solutions.
Or is deleting the pull request the only solution and start again with a new one?

**Comment by kenjis** [comment]
> the output stays the same.

Do you mean the same error message?
If so, please delete the branch in your local repository, and fetch the branch in GitHub again,
and try.

I got your PR branch and tried and got:
```
$ git fetch upstream 
$ git rebase upstream/develop 
Successfully rebased and updated refs/heads/snelledre/develop.
```
I think you will got the same message.

Your branch has a merge commit, so `git rebase` will remove it.

**Comment by kenjis** [comment]
> I don't understand why so small thing is so hard to get right.

It seems you are a very beginner of Git.
Git is very difficult tool, so it is difficult to understand.
And if you don't understand Git concepts, you cannot use it easily.

Starting again new PR might be easier than solving your current issue.

If you create a new PR, I recommend you create a new branch for the PR,
not using `develop` branch.

**Comment by datamweb** [comment]
@snelledre Starting anything is a bit difficult, if you are not familiar with it, it becomes more difficult.

Your situation in this PR has existed for all of us one day, like me confused, ambiguous and :weight_lifting:

You are not the first person in this situation and you will not be the last :snake:

The conflict and difficulties of these PR can make you interested in learning new topics such as git, github cli ,phpunit test and... think positive 🦾

As @kenjis' said before, if you don't succeed, just create a new PR.

**Comment by kenjis** [comment]
If you don't know how to use git, I recommend you try https://learngitbranching.js.org/

**Pull request closed without merging**